### PR TITLE
new release note for cloud soar

### PR DESCRIPTION
**Pull request by fzampori**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: blog-csoar/2024-03-26-application-update.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: March 26, 2024 - Application Update
+keywords:
+  - sumo logic
```
Comment (kimsauce) [inline blog-csoar/2024-03-26-application-update.md:4]:
```suggestion
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
One fix (remove sumo Logic as keyword; too generic).

File: blog-csoar/2024-03-26-application-update.md (hunk 1 above)
Comment (fzampori) [reply blog-csoar/2024-03-26-application-update.md:4]: I see, but that keyword is already present in other content 🤔

**Review by jpipkin1 — APPROVED** [review]
Approved, @fzampori. I made a few changes (see commit "[Updates per review](https://github.com/SumoLogic/sumologic-documentation/pull/3891/commits/81cc0aef259cc5b3e8c500de6c938cbf6d2aae5c)").

**Comment by jpipkin1** [comment]
@kimsauce, could you approve this PR?

**Review by kimsauce — APPROVED** [review]

**Pull request merged**